migrations/versions/48b78978df12_add_embedding_column_to_messages_table.py

```
@@ -0,0 +1,55 @@
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+schema = settings.DB.SCHEMA
+
+
+def upgrade() -> None:
+    op.add_column("messages", sa.Column("embedding", Vector(1536), nullable=True))
+    try:
+        op.execute(
+            text(
+                f"""
+                CREATE INDEX idx_messages_embedding_hnsw ON {schema}.messages 
```

Flake8 (7.2.0)

[warning] trailing whitespace

(W291)